### PR TITLE
Get rid of warnings/errors spam in build

### PR DESCRIPTION
**Pull request by dcolligan**
Issue #780

### CODE REVIEW

File: tools/sphinx/protobuf-uml.py [hunk 1]
```diff
@@ -181,10 +181,8 @@ def write_graph(fields, containments, nests, matched_references, clusters):
 
     # Define edge properties for containments
     graph += "edge [\n"
-    graph += "\tdir=both\n"
```
Comment (david4096) [inline tools/sphinx/protobuf-uml.py]: What's happening here? Are they ignored values?

Comment (dcolligan) [inline tools/sphinx/protobuf-uml.py]: yeah, I deleted everything that the warning output said it was ignoring

**Comment by dcolligan** [comment]
@david4096  If one wants to see errors / warnings from the sphinx build, can't one just turn them back on?  Errors and warnings are no good if everyone habitually ignores them.

**Comment by david4096** [comment]
Add a flag that makes it easy for the current logging to happen to change the default. I think straight passing down the arguments to `make docs` might be fastest/easiest, but if you want to add a `verbose` key, no argument here.

**Comment by dcolligan** [comment]
@david4096 flag added.  Get the warning output by doing: `make docs verbose=true`

**Comment by david4096** [comment]
Thanks, we don't deserve to take credit for all those warnings when we're trying to parse other people's comments :).

**Pull request merged**